### PR TITLE
R 3.5.0 for iomkl 2018a

### PR DESCRIPTION
**Pull request by edmondac**
Here's our R 3.5.0 using iomkl 2018a. I don't know if you'd rather this came with 2018.02...?

It's still building here to test, but I wanted to ask that question sooner rather than later!

This PR is a prerequisite for Bioconductor 3.7, which has at least one library that requires R 3.5.0.

### CODE REVIEW

**Comment by boegel** [comment]
@edmondac R 3.5.0 is our target for `foss/2018b` and `intel/2018b` (cfr. #6424 & #6409 resp.), but we should also update the (dependency and) extension versions when doing that...

**Comment by edmondac** [comment]
Hi @boegel - so what would you like me to do with this PR?

**Comment by boegel** [comment]
@edmondac I think we should either update the extensions as well, or abandon this...

File: easybuild/easyconfigs/g/GEOS/GEOS-3.6.2-iomkl-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+easyblock = 'ConfigureMake'
+
+name = 'GEOS'
+version = '3.6.2'
+
+homepage = 'http://trac.osgeo.org/geos'
+description = """GEOS (Geometry Engine - Open Source) is a C++ port of the  Java Topology Suite (JTS)"""
+
+toolchain = {'name': 'iomkl', 'version': '2018a'}
```
[the remark this answers is not in the document]
Comment (edmondac) [reply easybuild/easyconfigs/g/GEOS/GEOS-3.6.2-iomkl-2018a.eb]: Good point - removed.

File: easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-iomkl-2018a-Mesa-17.3.6.eb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'libGLU'
+version = '9.0.0'
+versionsuffix = '-Mesa-17.3.6'
```
[the remark this answers is not in the document]
Comment (edmondac) [reply easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-iomkl-2018a-Mesa-17.3.6.eb]: Sorry - missed that one. I'd changed the dep but not removed the file.

File: easybuild/easyconfigs/g/GEOS/GEOS-3.6.2-iomkl-2018a.eb (hunk 1 above)
[the remark this answers is not in the document]
Comment (edmondac) [reply easybuild/easyconfigs/g/GEOS/GEOS-3.6.2-iomkl-2018a.eb]: I updated using easy_update, then a little tinkering.

[the remark this answers is not in the document]
Comment (edmondac) [reply easybuild/easyconfigs/g/GEOS/GEOS-3.6.2-iomkl-2018a.eb]: It's still building here with the updated apps, it's about half way through them...

File: easybuild/easyconfigs/r/R/R-3.5.0-iomkl-2018a-X11-20180131.eb [hunk 3]
```diff
@@ -0,0 +1,1874 @@
...
+        'checksums': ['f524148fdb29aac3a178618f88718d3d4ac91283014091aa11a01f1c70cd4e51'],
+    }),
+    ('GGally', '1.4.0', {
+        'checksums': ['9a47cdf004c41f5e4024327b94227707f4dad3a0ac5556d8f1fba9bf0a6355fe'],
+    }),
+    ('beanplot', '1.2', {
+        'checksums': ['49da299139a47171c5b4ccdea79ffbbc152894e05d552e676f135147c0c9b372'],
+    }),
+    ('clValid', '0.6-6', {
+        'checksums': ['bcf781238a2a831a7ec8909d27251731e3115a0541a9ea9795e5ec4d903938d1'],
```
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.5.0-iomkl-2018a-X11-20180131.eb]: @edmondac I'm getting a different checksum here: `c13ef1b6258e34ba53615b78f39dbe4d8ba47b976b3c24a3eedaecf5ffba19ed` (fresh download)...

File: easybuild/easyconfigs/r/R/R-3.5.0-iomkl-2018a-X11-20180131.eb [hunk 4]
```diff
@@ -0,0 +1,1874 @@
...
+    }),
+    ('sna', '2.4', {
+        'checksums': ['2292b3190e8d879e494527ae9d9d1174c31cb4183749ecb455aedd8d534048cf'],
+    }),
+    ('glasso', '1.8', {
+        'patches': [('glasso-1.8-ifort-no-fixed.patch', 1)],
+        'checksums': [
+            'fc9ba74519388c194e14bae39d41f3fec8c2aade0177df364f61753f0fbf8a3d',  # glasso_1.8.tar.gz
+            '87e966b6651c2d9ccd8a2c2ba842eef8dcbeaa3fdd25085bb1d5aff6e79f409c',  # glasso-1.8-ifort-no-fixed.patch
+
```
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.5.0-iomkl-2018a-X11-20180131.eb]: this patch no longer applies...

Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.5.0-iomkl-2018a-X11-20180131.eb]: `gee` extension is missing (cfr. #6376)

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2002.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/46005a3cc840cdad98648b4a38820339 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2012.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3c1159f2af875928c4b81f7959d4e3b1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3236.victini.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/7ac0721c01771859ee401d50cd50fc12 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @edmondac!

**Pull request merged**